unittests/snapshot_tests.cpp

```diff
@@ -663,11 +663,15 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_kv_snapshot, SNAPSHOT_SUITE, snapshot_suites)
    //for (bool rocks_save : { false, true }) {
    //   for (bool rocks_load : { false, true }) {
 
+   // snapshot handling is being rewritten.
```

Comment:
put a #warning here so that we don't forget to turn it back on

Reply:
Will do.